Binance Detect Moonings.py

```diff
@@ -378,9 +380,16 @@ def sell_coins():
             continue
 
         # check that the price is below the stop loss or above take profit (if trailing stop loss not used) and sell if this is the case
-        if LastPrice < SL or LastPrice > TP and not USE_TRAILING_STOP_LOSS:
-            print(f"{txcolors.SELL_PROFIT if PriceChange >= 0. else txcolors.SELL_LOSS}TP or SL reached, selling {coins_bought[coin]['volume']} {coin} - {BuyPrice} - {LastPrice} : {PriceChange-(TRADING_FEE*2):.2f}% Est:${(QUANTITY*(PriceChange-(TRADING_FEE*2)))/100:.2f}{txcolors.DEFAULT}")
-
+        if (SELL_AFTER_DURATION > 0 and HoldTime >= SELL_AFTER_DURATION) or LastPrice < SL or LastPrice > TP and not USE_TRAILING_STOP_LOSS:
+            if LastPrice < SL:
+                print(f"{txcolors.SELL_PROFIT if PriceChange >= 0. else txcolors.SELL_LOSS}SL reached, selling {coins_bought[coin]['volume']} {coin} - {BuyPrice} - {LastPrice} : {PriceChange-(TRADING_FEE*2):.2f}% Est:${(QUANTITY*(PriceChange-(TRADING_FEE*2)))/100:.2f} Held for {int(HoldTime)} minutes {txcolors.DEFAULT}")
+            if LastPrice > TP:
```

Comment:
Why would you want to sell during take profit. Also, this functionality will ONLY execute if TSL is off. Almost everyone uses TSL...